src/test/java/posters/cucumber/support/ProductSupport.java

```diff
@@ -3,19 +3,19 @@
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import posters.pageobjects.pages.browsing.CategoryPage;
-import posters.pageobjects.pages.browsing.ProductdetailPage;
+import posters.pageobjects.pages.browsing.ProductDetailPage;
 
 public class ProductSupport
 {
     @When("^product \"([^\"]*)\" is opened$")
-    public ProductdetailPage clickProductByName(String productName)
+    public ProductDetailPage clickProductByName(String productName)
     {
         return new CategoryPage().clickProductByName(productName);
     }
 
     @Then("^a product detail page shows the headline \"([^\"]*)\"$")
     public void validateProductName(String name)
     {
-        new ProductdetailPage().validateProductName(name);
+        new ProductDetailPage().validateProductName(name);
```

Comment:
is a return type needed at this place?
In general the Steps are not able to communicate with each other on this level, so please remove the return if it'n unused.

Reply:
please check the other Support classes regarding this topic